.github/workflows/post-merge.yml

```diff
@@ -42,12 +42,10 @@ jobs:
     strategy:
```

Comment:
Given that we have also to support 22.04 for this release before we deprecate it entirely I'd actually argue that we could keep the "runner_version" parameter.

Reply:
We could but we would need to have a way to generate and store two versions of the packages, one for 24.04 and another for 22.04. I'll look into this